galsim/interpolatedimage.py

```diff
@@ -638,24 +679,24 @@ def __init__(self, real_kimage, imag_kimage, k_interpolant=None, stepk=None,
             raise ValueError("Real and Imag kimages must have same scale.")
 
         # Make sure any `wcs`s are `PixelScale`s.
-        if ((real_kimage.wcs is not None
-             and not real_kimage.wcs.isPixelScale())
-            or (imag_kimage.wcs is not None
-                and not imag_kimage.wcs.isPixelScale())):
+        if ((real_kimage.wcs is not None and
+             not real_kimage.wcs.isPixelScale()) or
+            (imag_kimage.wcs is not None and
+                not imag_kimage.wcs.isPixelScale())):
             raise ValueError("Real and Imag kimage wcs's must be PixelScale's or None.")
 
         # Check for Hermitian symmetry properties of real_kimage and imag_kimage
         shape = real_kimage.array.shape
         # If image is even-sized, ignore first row/column since in this case not every pixel has
         # a symmetric partner to which to compare.
-        bd = galsim.BoundsI(real_kimage.xmin + (1 if shape[1]%2==0 else 0),
+        bd = galsim.BoundsI(real_kimage.xmin + (1 if shape[1] % 2 == 0 else 0),
                             real_kimage.xmax,
-                            real_kimage.ymin + (1 if shape[0]%2==0 else 0),
+                            real_kimage.ymin + (1 if shape[0] % 2 == 0 else 0),
```

Comment:
Alright.  You've convinced me to drop the flake8.  I think I've reverted most of the style changes now.